src/shogun/lib/parameter_observers/ParameterObserverLogger.h

```diff
@@ -0,0 +1,40 @@
+/*
+* Written (W) 2019 Giovanni De Toni
+*/
+
+#ifndef SHOGUN_PARAMETEROBSERVERLOGGER_H
+#define SHOGUN_PARAMETEROBSERVERLOGGER_H
+
+#include <shogun/base/SGObject.h>
+#include <shogun/lib/parameter_observers/ParameterObserverInterface.h>
+
+namespace shogun {
+	/**
+	 * This class implements a logger which will basically print on screen all
```

Comment:
"This class implements a logger which prints all observed updates" .. much simpler